update-xfce-config

```diff
@@ -28,6 +36,16 @@ if [ "$UPDATE_CONFIG_DONE" -lt 1 ]; then
         "/commands/custom/<Control><Alt><Shift>P" \
         "qvm-unpause --all"
 fi
+if [ "$UPDATE_CONFIG_DONE" -lt 2 ]; then
+    reset_xfconf_if_default \
+        "xfce4-keyboard-shortcuts" \
+        "/commands/custom/<Control><Alt>Esc" \
```

Comment:
This fails for me with:
```
Property "/commands/custom/<Control><Alt>Esc" does not exist on cannel "xfce4-keyboard-shortcuts".
```
Looking at `xfconf-query -c xfce4-keyboard-shortcuts` output, I do have `/commands/default/<Control><Alt>Escape` (note `custom` -> `default` and `Esc` -> `Escape`). But then, if I manually add `qvm-xkill` command, I get extra property `/commands/custom/<Control><Alt>Escape` (but the one with "default" remains, with value `xkill`).
I'm not really sure what is the proper way to handle this (whether to modify default or to add custom).

Reply:
I believe we should modify the default, so that it will be reflected if the user ever reverts to the default configuration.